src/components/capture/creation/index.tsx

```diff
@@ -58,48 +70,40 @@ function NewCaptureModal() {
 
     // Form Event Handlers
     const handlers = {
-        close: () => {
-            navigate('..'); //This is assuming this is a child of the /captures route.
-        },
-
-        delete: () => {
-            console.log('Delete? You sure?');
-        },
-
-        save: (event: MouseEvent<HTMLElement>) => {
+        addToChangeSet: (event: MouseEvent<HTMLElement>) => {
             event.preventDefault();
-            let catalogVal = '';
 
-            if (editorRef.current) {
-                catalogVal = editorRef.current.getValue();
-            }
+            const namespace: string = details.data.name;
 
-            setFormSubmitting(true);
+            // TODO: Get user identifier from authentication-related application state.
+            const capture: Entity = {
+                metadata: {
+                    changeType: 'New Entity',
+                    entityType: 'Capture',
+                    name: namespace.substring(
+                        namespace.lastIndexOf('/') + 1,
+                        namespace.length
+                    ),
+                    namespace,
+                    user: 'temp@gmail.com',
+                },
+                schema: schemaFromEditor || catalogResponse,
```

Comment:
Fallback on the schema received from the discovery call in the event there is any issue retrieving the schema presented in the editor. We should discuss how this event should be handled from a UX perspective.

**Initial UX Thoughts:**
Displaying a page-level error message describing the error and the fallback option seems necessary to me. It would be great if an opportunity to re-edit the base schema (i.e., that returned by the discovery call) could be offered as well.

Reply:
Yeah - that's what should happen.